board/flags-config.cmake

```diff
@@ -0,0 +1,66 @@
+# This function provides a set of specific flags for each supported board
+# Depending on the processor type. The objective is to optimize for size.
+# Thus, all of the fllowing flags are chosen carefully to reduce binary 
```

Comment:
```suggestion
# Thus, all of the following flags are chosen carefully to reduce binary 
```

Reply:
I really need a good keyboard, a true difference when using a laptop keyboard and my dell PC